WcaOnRails/config/routes.rb

```diff
@@ -108,6 +108,7 @@
   get 'logo' => 'static_pages#logo'
   get 'wca-workbook-assistant' => 'static_pages#wca_workbook_assistant'
   get 'wca-workbook-assistant-versions' => 'static_pages#wca_workbook_assistant_versions'
+  get 'organizer_guidelines' => 'static_pages#organizer_guidelines'
```

Comment:
I personally prefer dashes to underscores in urls, so `organizer-guidelines` instead.